file-tracker/file_tracker/client_connection.py

```diff
@@ -40,8 +40,11 @@ async def on_message(message):
                         response.message = ls(self.path)
 
                     elif message.startswith("tail:"):
-                        filename = message.split(":")[1]
-                        response.message = tail(self.path, filename)
+                        _, args = message.split(":", 1)
+                        args = args.split(",")
+                        filename = args[0]
+                        n_lines = int(args[1]) if len(args) > 1 else 10
```

Comment:
Is the validation of `n_lines` done on the client? Nevertheless, I think it is better to add a `try-catch` to validate if `args[1]` is an integer and provide an appropriate error message if it isn't.